components/alert.css

```diff
@@ -0,0 +1,60 @@
+.alert-container {
+  width: 90%;
+}
+.alert {
+  width: 100%;
+  padding: 1.5rem;
+  position: relative;
+  border-left: 0.8rem solid #bababa;
+  background: #ededed;
+  color: #a1a1a1;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  text-align: left;
+  font-size: 1.6rem;
+  align-content: center;
+  flex-wrap: nowrap;
+  margin-bottom: 1rem;
+  margin-left: 1rem;
+  font-weight: bold;
+}
+.alert .alert-icon {
+  margin-right: 1rem;
+}
+.alert .alert-content {
+  width: 100%;
+  font-weight: bolder;
+}
+.alert .alert-close {
+  margin-left: 1rem;
+}
+
```

Comment:
Use consistent formatting in your file.

Reply:
I have prettier installed and code is formatted . Can you explain this please.